include/SDL3/SDL_events.h

```diff
@@ -1565,6 +1565,38 @@ extern SDL_DECLSPEC Uint32 SDLCALL SDL_RegisterEvents(int numevents);
  */
 extern SDL_DECLSPEC SDL_Window * SDLCALL SDL_GetWindowFromEvent(const SDL_Event *event);
 
+/**
+ * Generate a human-readable description of an event.
+ *
+ * This will fill `buf` with a null-terminated string that might look
+ * something like this:
+ *
+ * ```
+ * SDL_EVENT_MOUSE_MOTION (timestamp=1140256324 windowid=2 which=0 state=0 x=492.99 y=139.09 xrel=52 yrel=6)
+ * ```
+ *
+ * The exact format of the string is not guaranteed; it is intended for
+ * logging purposes, to be read by a human, and not parsed by a computer.
+ *
+ * The returned value follows the same rules as SDL_snprintf(): `buf`
+ * will always be NULL-terminated (unless `buflen` is zero), and will be
+ * truncated if `buflen` is too small. The return code is the number of bytes
+ * needed for the complete string, not counting the NULL-terminator, whether
+ * the string was truncated or not. Unlike SDL_snprintf(), though, this
+ * function never returns -1.
+ *
+ * \param event an event to describe. May be NULL.
+ * \param buf the buffer to fill with the description string. May be NULL.
+ * \param buflen the maximum bytes that can be written to `buf`.
+ * \returns number of bytes needed for the full string, not counting the
+ *          null-terminator byte.
+ *
+ * \threadsafety It is safe to call this function from any thread.
+ *
+ * \since This function is available since SDL 3.2.0.
```

Comment:
```suggestion
 * \since This function is available since SDL 3.4.0.
```

Reply:
I think build-scripts/fnsince.pl corrects these when the wiki bridge runs, fwiw, but better safe than sorry.